templates/gatsby-drupal/files/drupal/README.md

```diff
@@ -0,0 +1,42 @@
+# Drupal 8 for Platform.sh
```

Comment:
This file seems redundant.  Or do we actually need it?  (We should be consistent with other multi-app templates, whatever we do.)